src/jgikbase/idmapping/userhandlers/kbase_user_handler.py

```diff
@@ -50,9 +51,11 @@ def _check_error(self, r):
                               str(r.status_code))
             # assume that if we get json then at least this is the auth server and we can
             # rely on the error structure.
+            if j['error']['appcode'] == 10020:
```

Comment:
Is there a way to use an exception class name or a string identifier here instead? Having to cross reference a number without intrinstic meaning is not super readable

Reply:
Looks like this in my local branch now:

```
            if j['error']['apperror'] == 'Invalid token':
                raise InvalidTokenError('KBase auth server reported token is invalid.')
```